blazor/datagrid/connecting-to-database/microsoft-sql-server.md

```diff
@@ -233,7 +335,7 @@ public object Post([FromBody] DataManagerRequest DataManagerRequest)
 }
 {% endhighlight %}
 
-### Handling grouping operation
+<!--### Handling grouping operation
```

Comment:
why it is commented

Reply:
While Using UrlAdaptor, grouping actions will be executed in client side. can we remove this part from documentation?